.travis.yml

```diff
@@ -98,7 +98,10 @@ jobs:
       - docker build -t populate . # <-- Temporary solution until populate image is updated. Or maybe we keep.
       - cd -
       # Install Customer <-- Reminder to update the load_data.yaml to point to agreed populate image later
+      - helm dependency update ./chart/customer
       - helm install --set travis=true --set service.minikubeIp=$MINIKUBE_IP --set service.authKubePort=$AUTH_NODE_PORT --set image.repository=$IMAGE_NAME --set image.tag=$IMAGE_TAG --name $RELEASE_NAME ./chart/customer/
+      # Wait for CouchDB to be installed and populated from Customer installation
+      - sleep 60
```

Comment:
Can use what we have in Orders for waiting for the populate job to complete
https://github.com/ibm-cloud-architecture/refarch-cloudnative-micro-orders/blob/8b81ddd93045556b728f5fb6e7f8f704e85a91ce/.travis.yml#L110-L112